README.md

```diff
@@ -8,6 +8,97 @@ software below.
+  Run the following lines in shell:
+    wget -q -O - https://www.lrde.epita.fr/repo/debian.gpg | apt-key add -
+    echo 'deb http://www.lrde.epita.fr/repo/debian/ stable/' >> /etc/apt/sources.list
+    apt-get update
+    apt-get install spot libspot-dev spot-doc python3-spot # Or a subset of those
+
+3) Install necessary dependencies:
+  Run the following lines in shell:
+    apt install python-pip
+    pip2 install matplotlib
+    pip2 install numpy
+    apt-get install python-tk
+    pip2 install networkx==1.11
+    pip2 install pyyamlCollecting pyyaml
+    pip2 install pp
+    apt install cmake
+
+    (Note: ensure that Python 2.7 is installed)
+
+  4) Download and unpack ltl2dstar
+    Download from: https://www.ltl2dstar.de/
+    Navigate to the ltl2dstar folder
+    Run the following lines in shell:
+      mkdir build
+      cd build/
+      cmake -DCMAKE_BUILD_TYPE=Release ../src
+      make
+      less ~/.bashrc
+      mkdir ~/bin
+      cp ltl2dstar ~/bin/
+      echo 'export PATH="$PATH:$HOME/bin"' >> ~/.bashrc
+
+  5) Set $PYTHONPATH to include the location of the lomap library:
+    Run the following line in shell:
+      export PYTHONPATH="${PYTHONPATH}:/path/to/lomap"
+
+  6) Test if the setup worked properly:
+    Navigate to /lomap/lomap/tests
+    Run any of the Python test files
+      Ex. python test_automata.py
+
+  Common Issues:
+    1) ImportError: No module named lomap.classes
+      Problem: The lomap library is not in the path variable
+      Possible Solution: Manually add lomap to your Python directory
+
+    2)  File "/usr/bin/pip", line 9, in <module>
+        from pip import main
+        ImportError: cannot import name main
+          Problem: Wrong version of pip (are using Python 3 pip)
+          Solution: Run commands with pip2
+
+    3) AttributeError: 'Graph' object has no attribute 'nodes_iter' (or other graph issues)
+        Problem: Wrong version of networkx installed
+        Solution: Uninstall networkx and install networkx1.11 (see above)
+
+
+----------------------------------------------------------------------
+
+UROP 2018-19 Todo List:
+- Allow usage of nx 2.x
+- Allow usage of Python 3.x
+- Remove Old Ts/Markov file format system, use yaml
+- Revise installation instructions
+- Revise developer instructions, add more tests
+- Create more testing examples with instructions (docs)
+- Improve visualization
+- Implement dfsa and dfsca minimization
+- Add support for DFSCAs
+- Integrate logic minimization
+- Add Buchi/Rabin games
+- Sync with twtl, reactive-lfl, lurmod, purp, golfe-firne
+- Integrate GDTL and predicate system
+- Upgrade GDTL/TWTL grammars to ANTLRv4
+- Add more examples
+- Add RH/MPC framework
+- General clean-up of code
+- Test for functions and continuous integration system
+- BDDs for guards
+
+----------------------------------------------------------------------
+
 LTL Optimal Multi-Agent Planner (LOMAP)
 Copyright (C) 2012-2015, Alphan Ulusoy (alphan@bu.edu)
 
```

Comment:
Make a new section called "Copyright and warranty information" that includes _"This program [...] called 'COPYING'."_.
Also, change "COPYING" to "license.txt", delete the file `COPYING`, and rename `LICENSE` to `license.txt`.